AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC.xcodeproj/project.pbxproj

```diff
@@ -884,6 +954,7 @@
 		};
 		1D992FF3231FA1C500C472F8 /* Release */ = {
 			isa = XCBuildConfiguration;
+			baseConfigurationReference = 183940DA84D0E994AA208ACB /* Pods-Non-Mediated Projects-AppLovin MAX Demo App - ObjC.release.xcconfig */;
```

Comment:
We shouldn't check in any CocoaPods project config changes - project should be vanilla until pub runs CocoaPods.